flexget/plugins/input/next_trakt_episodes.py

```diff
@@ -75,6 +75,7 @@ def on_task_input(self, task, config):
                     trakt_id = item['show']['ids']['trakt']
                     listed_series[trakt_id] = {
                         'series_name': '%s (%s)' % (item['show']['title'], item['show']['year']),
+                        'series_year': item['show']['year'],
```

Comment:
Well, we have more of a built in concept of year for the movie stuff, and the offline parser fills it. With series, due to historical reasons, we have not had an explicit year involved, it's always just been lumped in in the series title.